### DISCUSSION TITLE
Should the editing mode names be in full by default? (e.g. INS becomes INSERT)

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#4589] [Polls]
```
[editor.statusline]
mode.normal = "NORMAL" # from NOR
mode.insert = "INSERT" # from INS
mode.select = "SELECT" # from SEL
```
In Vim and Neovim insert mode is `-- INSERT --`, even longer. In Helix the abbreviated versions made me think twice about what they meant, and also are not any well know abbreviation, so they don't really make any sense. I would vote to make them the full word.

**Comment by archseer** [comment]
I prefer to have these as small as possible, and the original design had these as symbols or single letters.

In vim and neovim the mode displays in the command buffer below the statusline where there's plenty of space, but in helix this is a part of the statusline where every character saved gives you more space to display something else. By coincidence the mode names are all the same length right now but if we added new ones (or renamed current ones) then the difference in width would cause elements on the statusline to shift as the modes changed.
